src/app/core/facades/theme.facade.ts

```diff
@@ -0,0 +1,58 @@
+import { DOCUMENT } from '@angular/common';
+import { Inject, Injectable, Renderer2, RendererFactory2 } from '@angular/core';
+import { Store, select } from '@ngrx/store';
+import { BehaviorSubject } from 'rxjs';
+import { distinctUntilChanged } from 'rxjs/operators';
+
+import { THEMES } from 'ish-core/configurations/injection-keys';
+import { Theme } from 'ish-core/models/theme/theme.model';
+import { getTheme } from 'ish-core/store/configuration';
+
+@Injectable({
+  providedIn: 'root',
+})
+export class ThemeFacade {
```

Comment:
this is not really a facade, it is more something like a utility service. please rename and move it